### PR TITLE
[INLONG-9960][Manager] Support to config kafka data node

### LINKED ISSUE #9960: [Feature][Manager] Manager support to config Kafka data node
### Description

Manager support to config Kafka data node

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by castorqin**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #9960 

### Motivation

 Support to config kafka data node

### Modifications
The details can be found at [#9960](https://github.com/apache/inlong/issues/9960).

### CODE REVIEW

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+
+    @Override
+    public Boolean testConnection(DataNodeRequest request) {
+        KafkaDataNodeRequest kafkaDataNodeRequest = (KafkaDataNodeRequest) request;
+        String bootstrapServers = kafkaDataNodeRequest.getBootstrapServers();
+        Preconditions.expectNotBlank(bootstrapServers, ErrorCodeEnum.INVALID_PARAMETER,
+                "connection bootstrapServers  cannot be empty");
+        if (getKafkaConnection(bootstrapServers)) {
+            LOGGER.info("kafka  connection success for bootstrapServers={}",
```
Comment (vernedeng) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java:101]:
```suggestion
            LOGGER.info("kafka connection success for bootstrapServers={}",
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    }
+
+    /**
+     * Get the dto instance from the JSON string.
+     */
+    public static KafkaDataNodeDTO getFromJson(@NotNull String extParams) {
+        try {
+            return JsonUtils.parseObject(extParams, KafkaDataNodeDTO.class);
+        } catch (Exception e) {
+            throw new BusinessException(ErrorCodeEnum.GROUP_INFO_INCORRECT,
```
Comment (fuweng11) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java:71]: DATA_NODE_INFO_INCORRECT

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    @Override
+    protected void setTargetEntity(DataNodeRequest request, DataNodeEntity targetEntity) {
+        KafkaDataNodeRequest nodeRequest = (KafkaDataNodeRequest) request;
+        CommonBeanUtils.copyProperties(nodeRequest, targetEntity, true);
+        try {
+            KafkaDataNodeDTO dto = KafkaDataNodeDTO.getFromRequest(nodeRequest, targetEntity.getExtParams());
+            targetEntity.setExtParams(objectMapper.writeValueAsString(dto));
+        } catch (Exception e) {
+            throw new BusinessException(ErrorCodeEnum.SOURCE_INFO_INCORRECT,
```
Comment (fuweng11) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java:89]: DATA_NODE_INFO_INCORRECT

**Review by aloyszhang — APPROVED** [review]

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java (hunk 1 above)
Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java]: done

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java (hunk 2 above)
Comment (castorqin) [reply inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java (hunk 3 above)
Comment (castorqin) [reply inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/node/kafka/KafkaDataNodeOperator.java]: done

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/ErrorCodeEnum.java [hunk 4]
```diff
@@ -73,6 +73,7 @@ public enum ErrorCodeEnum {
     DATA_NODE_NOT_FOUND(1150, "Data node information does not exist"),
     DATA_NODE_TYPE_NOT_SUPPORTED(1151, "Data node type '%s' not supported"),
     DATA_NODE_ID_CHANGED(1152, "Data node information's id not equals"),
+    DATA_NODE_INFO_INCORRECT(1153, "data node info was incorrect"),
```
Comment (dockerzhang) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/enums/ErrorCodeEnum.java:76]:
```suggestion
    DATA_NODE_INFO_INCORRECT(1153, "Data node info was incorrect"),
```

**Review by fuweng11 — APPROVED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/kafka/KafkaClusterRequest.java [hunk 5]
```diff
@@ -36,6 +38,10 @@
 @ApiModel("Inlong cluster request for Kafka")
 public class KafkaClusterRequest extends ClusterRequest {
 
+    @JsonProperty("bootstrap.servers")
+    @ApiModelProperty(value = "Kafka bootstrap servers' URL, is the 'url' field of the cluster")
```
Comment (dockerzhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/kafka/KafkaClusterRequest.java:42]:
```suggestion
    @ApiModelProperty(value = "Kafka bootstrap servers' URL")
```

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/kafka/KafkaClusterInfo.java [hunk 6]
```diff
@@ -39,6 +41,10 @@
 @ApiModel("Inlong cluster info for Kafka")
 public class KafkaClusterInfo extends ClusterInfo {
 
+    @JsonProperty("bootstrap.servers")
+    @ApiModelProperty(value = "Kafka bootstrap servers' URL, is the 'url' field of the cluster")
```
Comment (dockerzhang) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/cluster/kafka/KafkaClusterInfo.java:45]:
```suggestion
    @ApiModelProperty(value = "Kafka bootstrap servers' URL")
```

**Review by dockerzhang — DISMISSED** [review]

**Review by fuweng11 — APPROVED** [review]

File: inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+ * Kafka data node info
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@ApiModel("Kafka data node info")
+public class KafkaDataNodeDTO {
+
+    @ApiModelProperty("kafka bootstrapServers")
```
Comment (vernedeng) [inline inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/node/kafka/KafkaDataNodeDTO.java:45]: Lack of audit_set_name params

**Review by vernedeng — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**